### ISSUE TITLE
ManagementCenterServiceTest runScript

### ISSUE THREAD

**Issue by lazerion** [hazelcast/hazelcast#17840]
- Fails on `Hazelcast-4.maintenance-ZuluJDK15`
- Fails on [Build #3 (Nov 8, 2020 7:30:42 AM)](http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-4.maintenance-ZuluJDK15/3/testReport/junit/com.hazelcast.client.management/ManagementCenterServiceTest/runScript/)
- Error
```
java.lang.IllegalArgumentException: Could not find ScriptEngine named 'javascript'.
```
- Stacktrace
```
Caused by: java.lang.IllegalArgumentException: Could not find ScriptEngine named 'javascript'.
	at com.hazelcast.internal.management.operation.RunScriptOperation.lambda$run$0(RunScriptOperation.java:65)
	at com.hazelcast.internal.util.executor.CompletableFutureTask.run(CompletableFutureTask.java:64)
	at com.hazelcast.internal.util.executor.CachedExecutorServiceDelegate$Worker.run(CachedExecutorServiceDelegate.java:217)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.lang.Thread.run(Thread.java:832)
	at com.hazelcast.internal.util.executor.HazelcastManagedThread.executeRun(HazelcastManagedThread.java:64)
	at com.hazelcast.internal.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:80)
	at ------ submitted from ------.()
	at com.hazelcast.internal.util.ExceptionUtil.cloneExceptionWithFixedAsyncStackTrace(ExceptionUtil.java:265)
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.returnOrThrowWithGetConventions(InvocationFuture.java:112)
```
- kindly check

**Comment by mmedenjak** [comment]
I guess it's because of https://openjdk.java.net/jeps/372 ?

**Comment by emre-aydin** [comment]
Fixed via #17857
